### DISCUSSION TITLE
iOS 16.3.1 doesnt connect

### DISCUSSION THREAD

**Discussion by dmitrygh** [mikebrady/shairport-sync#1719] [Q&A]
Hi!
Thank you much for your effort and perfect development!
A year ago on older iOS it worked perfectly, so a year later after update to iOS 16.3.1 and latest shairport 4.2.1 something goes wrong.

So, macOS connect to Airplay and play is ok, but 4-5 times resyncing from start before smooth play...
And, iOS 16.3.1 sees Airplay, trying to connect... and after app.15 seconds it says cannot connect... (macOS was disconnected first)
Also i tried installing --with-airplay-2 option, no changes, all is the same after reboot.

Please help!!! Its the only app to play lossless hi-fi!

**Comment by mikebrady** [comment]
Thanks for the kind words! I think this would be better handled as an Issue rather than a Discussion, so if you would open a Issue about it, we could take it up there.
